CloudM.Exercise.WebApi/Controllers/PackageController.cs

```diff
@@ -27,12 +31,63 @@ public async Task<PackageDependencies> Get(string name, string version)
                 PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
             });
 
+            var dependencies = npmPackage.Versions[version].Dependencies;
```

Comment:
It will be better if we add a null check so that we are safe if this version does not exist and if it does not exist to have a logic to return proper result